package.json

```diff
@@ -56,8 +56,8 @@
     "multer": "^1.4.2",
     "node-fetch": "3.2.6",
     "ncp": "2.0.0",
-    "passport": ">=0.4.1",
-    "passport-local": ">=1.0.0",
+    "passport": "0.6.0",
+    "passport-local": "1.0.0",
```

Comment:
Just noting that 1.0.0 is the only version and what was currently installed.
And passport 0.6.0 is also the current version.